app/src/main/java/org/openimis/imispolicies/ClientAndroidInterface.java

```diff
@@ -2263,6 +2268,20 @@ public JSONArray getRecordedPoliciesWithIdentifier(String InternalIdentifier) {
         return RecordedPolicies;
     }
 
+    public JSONArray getNotEnrolledPolicies(String insuranceNumber, String insuranceProduct, String renewal) {
+        String aRenewal = "";
+        if (!renewal.equals("")) {
+            aRenewal = " AND isDone == '"+renewal+"'";
+        }
+
+        String uploaded = " AND UploadedDate == ''";
```

Comment:
Why you decomposed the SQL in substrings and then you concatenate even if you don't have conditions?

Reply:
Yes, you're right. Without any conditions these additional variables are not needed and can be included directly in query string